### PR TITLE
Adding docs for setDelegationParameters()

### PR DESCRIPTION
**Pull request by MichaelMacaulay**

### CODE REVIEW

File: website/pages/en/network/indexing.mdx [hunk 1]
```diff
@@ -790,6 +790,26 @@ Once an Indexer has staked GRT in the protocol, the [Indexer components](/networ
 setDelegationParameters(950000, 600000, 500)
 ```
 
+### Setting delegation parameters
+
+The setDelegationParameters() function in the [staking contract](https://github.com/graphprotocol/contracts/blob/main/contracts/staking/Staking.sol) is essential for Indexers, allowing Indexers to set parameters that define their interactions with Delegators, influencing their reward sharing and delegation capacity.
```
Comment (benface) [inline website/pages/en/network/indexing.mdx:795]:
```suggestion
The `setDelegationParameters()` function in the [staking contract](https://github.com/graphprotocol/contracts/blob/main/contracts/staking/Staking.sol) is essential for Indexers, allowing them to set parameters that define their interactions with Delegators, influencing their reward sharing and delegation capacity.
```

File: website/pages/en/network/indexing.mdx [hunk 2]
```diff
@@ -790,6 +790,26 @@ Once an Indexer has staked GRT in the protocol, the [Indexer components](/networ
...
+
+This function enables an Indexer to define the following parameters:
+
+- indexingRewardCut: Percentage of indexing rewards retained for the Indexer.
+- queryFeeCut: Percentage of query fees retained by the Indexer.
+- \_cooldownBlocks: Period of time (in blocks) that need to pass to update delegation parameters.
```
Comment (benface) [inline website/pages/en/network/indexing.mdx:801]:
```suggestion
- `indexingRewardCut`: Percentage of indexing rewards retained for the Indexer.
- `queryFeeCut`: Percentage of query fees retained by the Indexer.
- `_cooldownBlocks`: Period of time (in blocks) that need to pass to update delegation parameters.
```

File: website/pages/en/network/indexing.mdx [hunk 3]
```diff
@@ -790,6 +790,26 @@ Once an Indexer has staked GRT in the protocol, the [Indexer components](/networ
...
+
+### How to set delegation parameters
+
+To set the delegation parameters using The Graphs Explorer interface, follow these steps:
```
Comment (benface) [inline website/pages/en/network/indexing.mdx:805]:
```suggestion
To set the delegation parameters using the Graph Explorer interface, follow these steps:
```

File: website/pages/en/network/indexing.mdx [hunk 4]
```diff
@@ -790,6 +790,26 @@ Once an Indexer has staked GRT in the protocol, the [Indexer components](/networ
...
+- `indexingRewardCut`: Percentage of indexing rewards retained for the Indexer.
+- `queryFeeCut`: Percentage of query fees retained by the Indexer.
+- `_cooldownBlocks`: Period of time (in blocks) that need to pass to update delegation parameters.
+
+### How to set delegation parameters
+
+To set the delegation parameters using the Graph Explorer interface, follow these steps:
+
+1. Navigate to [The Graph Explorer](https://thegraph.com/explorer/).
+2. Connect your wallet. Choose Gnosis Safe and then select Mainnet. Note: You will need to repeat this process for Arbitrum One.
```
Comment (azf20) [inline website/pages/en/network/indexing.mdx:808]: Why always a Gnosis safe?

File: website/pages/en/network/indexing.mdx [hunk 5]
```diff
@@ -790,6 +790,26 @@ Once an Indexer has staked GRT in the protocol, the [Indexer components](/networ
...
+3. Connect the wallet you have as a signer.
+4. Navigate to the 'Settings' section and select 'Delegation Parameters'. These parameters should be configured to achieve an effective cut within the range of approximately X-Y%. Upon entering values in the provided input fields, the interface will automatically calculate the effective cut. Adjust these values as necessary to attain the desired effective cut percentage.
```
Comment (azf20) [inline website/pages/en/network/indexing.mdx:810]: X-Y%? Is this intentionally a placeholder?

File: website/pages/en/network/indexing.mdx [hunk 6]
```diff
@@ -790,6 +790,26 @@ Once an Indexer has staked GRT in the protocol, the [Indexer components](/networ
...
 
+### Setting delegation parameters
+
+The `setDelegationParameters()` function in the [staking contract](https://github.com/graphprotocol/contracts/blob/main/contracts/staking/Staking.sol) is essential for Indexers, allowing them to set parameters that define their interactions with Delegators, influencing their reward sharing and delegation capacity.
+
+This function enables an Indexer to define the following parameters:
+
+- `indexingRewardCut`: Percentage of indexing rewards retained for the Indexer.
+- `queryFeeCut`: Percentage of query fees retained by the Indexer.
+- `_cooldownBlocks`: Period of time (in blocks) that need to pass to update delegation parameters.
```
Comment (azf20) [inline website/pages/en/network/indexing.mdx:801]: I think this is being changed soon @pcarranzav?

**Review by azf20 — CHANGES_REQUESTED** [review]
Some small questions

File: website/pages/en/network/indexing.mdx (hunk 6 above)
Comment (pcarranzav) [reply website/pages/en/network/indexing.mdx:801]: Yes, the Snapshot is up for the Council to vote, and we should be deploying this relatively soon

File: website/pages/en/network/indexing.mdx (hunk 5 above)
Comment (MichaelMacaulay) [reply website/pages/en/network/indexing.mdx:810]: Actually a typo - thank you for catching!

File: website/pages/en/network/indexing.mdx [hunk 7]
```diff
@@ -790,6 +790,20 @@ Once an Indexer has staked GRT in the protocol, the [Indexer components](/networ
...
+### Setting delegation parameters
+
+The `setDelegationParameters()` function in the [staking contract](https://github.com/graphprotocol/contracts/blob/main/contracts/staking/Staking.sol) is essential for Indexers, allowing them to set parameters that define their interactions with Delegators, influencing their reward sharing and delegation capacity.
+
+### How to set delegation parameters
+
+To set the delegation parameters using the Graph Explorer interface, follow these steps:
+
+1. Navigate to [The Graph Explorer](https://thegraph.com/explorer/).
+2. Connect your wallet. Choose multi-sig (such as Gnosis Safe) and then select mainnet. Note: You will need to repeat this process for Arbitrum One.
```
Comment (benface) [inline website/pages/en/network/indexing.mdx:802]:
```suggestion
2. Connect your wallet. Choose multisig (such as Gnosis Safe) and then select mainnet. Note: You will need to repeat this process for Arbitrum One.
```

**Review by benface — APPROVED** [review]

**Pull request merged**